.github/workflows/salesloft.yml

```diff
@@ -0,0 +1,75 @@
+name: SalesLoft
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+
+jobs:
+  rspec:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
```

Comment:
I think you can remove this postgres section, because Omniauth does not have a database